test/resources/get-pods-response.json

```diff
@@ -608,7 +608,7 @@
         "labels": {
           "name": "example-pod-v6",
           "controller-uid": "502b3b8b-cccf-4ac0-bbe2-642d13bdb7a9",
-          "job-name": "example-svc-v6"
+          "job-name": "example-job-1"
```

Comment:
I am not sure about this change. Why are we changing this test case?

Reply:
It was the only pod with a job-name label. Is the data hand-crafted?
The example is for example-job-1, and I expected the matching pod to have a matching job-name.

But I pushed a different version which instead adjusts the pod.